Certified Sail Extension (CSE) Examples/cse/appian-component-plugin.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<appian-component-plugin name="Map Component Plug-in" key="sample.map">
+  <plugin-info>
+    <description>Provides a Map Component Plug-in backed by Google Maps</description>
+    <vendor name="Appian" url="http://www.appian.com" />
+    <support supported="true" phone="703-442-8844" email="support@appian.com" url="https://community.appian.com"/>
```

Comment:
We should remove the phone number from the support info here (asked POs about this for Appian-supplied component plug-ins)